lib/src/cognito_user.dart

```diff
@@ -969,11 +982,34 @@ class CognitoUser {
     return true;
   }
 
-  /// This is used by authenticated users to enable MFA for him/herself
+  /// This is used by authenticated users to enable SMS-MFA for him/herself.
+  /// A verified phone number is required.
   Future<bool> enableMfa() async {
     if (_signInUserSession == null || !_signInUserSession!.isValid()) {
       throw Exception('User is not authenticated');
     }
+    
+    bool? phoneNumberVerified = false;
```

Comment:
@moonshinephoenix thank you for your contribution.
LGTM, just one nit - this variable can be non-nullable, correct? `bool phoneNumberVerified = false;`

Reply:
@furaiev Must have missed that. Fixed now